nix/machines/_common/base.nix

```diff
@@ -36,15 +36,44 @@
     tmux
     usbutils
     unixtools.nettools
-    vim
     wget
+    ((vim_configurable.override { }).customize {
+      name = "vim";
+      # Install plugins for example for syntax highlighting of nix files
+      vimrcConfig.packages.myplugins = with pkgs.vimPlugins; {
+        start = [
+          vim-nix
+          vim-lastplace
+        ];
+        opt = [ ];
+      };
+      vimrcConfig.customRC = ''
+        " Turn on syntax highlighting by default
+        syntax on
+        " Disable mouse
+        set mouse-=a
+      '';
+    })
   ];
 
   # Purge nano from being the default
   environment.variables = {
     EDITOR = "vim";
   };
 
+  # set 24h military time
+  i18n.extraLocaleSettings = {
```

Comment:
Can we remove all `LC_*` config and just leave the `LC_TIME`? `LC_TIME` is the only one that were concerned with at the moment